hack/lib/build/environment.sh

```diff
@@ -11,23 +11,24 @@ function os::build::environment::create() {
   local release_image="${OS_BUILD_ENV_IMAGE}"
   local additional_context="${OS_BUILD_ENV_DOCKER_ARGS:-}"
   if [[ "${OS_BUILD_ENV_USE_DOCKER:-y}" == "y" ]]; then
-    additional_context+="--privileged -v /var/run/docker.sock:/var/run/docker.sock"
+    additional_context+=" --privileged -v /var/run/docker.sock:/var/run/docker.sock"
+    local workingdir
+    workingdir=$( os::build::environment::release::workingdir )
```

Comment:
Workingdir has to create a container which is why it was in both if statements (because the third case we don't need to check it)